File: CS/CustomRibbonControl/CustomRibbonBarManager.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.XtraBars.Ribbon;
using DevExpress.XtraBars.ViewInfo;
using DevExpress.XtraBars.Ribbon.ViewInfo;
using DevExpress.XtraBars;

namespace LargeRibbonButton.CustomRibbonControl
{
    class CustomRibbonBarManager : RibbonBarManager
    {
        public CustomRibbonBarManager(RibbonControl ribbon) : base(ribbon) { }
        protected internal new BarDrawParameters DrawParameters { get { return PaintStyle.DrawParameters; } }
    }
}

File: VB/CustomRibbonControl/CustomRibbonBarManager.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports DevExpress.XtraBars.Ribbon
Imports DevExpress.XtraBars.ViewInfo
Imports DevExpress.XtraBars.Ribbon.ViewInfo
Imports DevExpress.XtraBars

Namespace LargeRibbonButton.CustomRibbonControl
    Friend Class CustomRibbonBarManager
        Inherits RibbonBarManager

        Public Sub New(ByVal ribbon As RibbonControl)
            MyBase.New(ribbon)
        End Sub
        Protected Friend Shadows ReadOnly Property DrawParameters() As BarDrawParameters
            Get
                Return PaintStyle.DrawParameters
            End Get
        End Property
    End Class
End Namespace
